documentation/book/con-kafka-mirror-maker-authentication.adoc

```diff
@@ -0,0 +1,87 @@
+// Module included in the following assemblies:
+//
+// assembly-kafka-mirror-maker-authentication.adoc
+
+[id='con-kafka-mirror-maker-authentication{context}']
+= Authentication support in Kafka Mirror Maker
+
+Authentication can be configured in the `authentication` sub-property of `consumer` and `producer` properties in `KafkaMirrorMaker.spec`.
```

Comment:
I wonder if "`KafkaMirrorMaker.spec.consumer.authentication` and `KafkaMirrorMaker.spec.producer.authentication` would be clearer? Describing it in a sentence as you have it ends up being backwards with the most nested object first.